lua/typescript-tools/protocol/text_document/completion/init.lua

```diff
@@ -65,14 +67,19 @@ function M.handler(request, response, params)
         sortText = "\u{ffff}" .. item.sortText
       end
 
+      local should_create_function_snippet = utils.should_create_function_snippet(kind, filetype)
+      local should_create_snippet = item.isSnippet or should_create_function_snippet
+      local label = is_optional and (item.name .. "?") or item.name
+      label = should_create_function_snippet and (label .. "(...)") or label
```

Comment:
I'm not sure I like this indicator, it clutters results too much for me. But, it is ok for now especially this feature is disabled by default. If cmp fixes its things can we go back only to tilda?

Reply:
Like I said it is ok for now, maybe cmp fixes this and we just fallback on `~`.